Jetcaster/app/src/main/java/com/example/jetcaster/ui/player/PlayerViewModel.kt

```diff
@@ -99,4 +100,14 @@ class PlayerViewModel @Inject constructor(
     fun onRewindBy(duration: Duration) {
         episodePlayer.rewindBy(duration)
     }
+
+    fun onAddToQueue() {
+        uiState.episodePlayerState.currentEpisode?.let {
+            episodePlayer.addToQueue(it)
+        }
+    }
+
+    fun onPlay(episode: PlayerEpisode) {
```

Comment:
curious why in `addToQueue` you don't have a parameter and use the `uiState.episodePlayerState.currentEpisode` while in `onPlay` you pass a parameter.

Reply:
Context: I was experimenting with the `MockEpisodePlayer#play(PlayerEpisode)` API to support swiping through episodes. But, I ended up not adding that feature to this PR since there are a couple of tricky edge cases. For that feature, you need to explicitly specify which episode you would like to play whereas the no-arg `onPlay` method is implicit (i.e. you want to play the current episode). 

For clarity, I can rename this to `onEpisodeChanged` or something like that. However, since this method is not used yet I'll remove it.